gather_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !js
```

Comment:
I am a bit confused about this build tag comment here. Whats the purpose of the `// +build` comment?

Reply:
Just to keep compatibility with older versions as the new directive was introduced in Go v1.7

Reply:
@mlsmaycon Thanks :)